### PR TITLE
Hide controls on tapping outside

### PR DESCRIPTION
**Pull request by rumaan**
toggle_controls

PS: Was lazy to get API keys 🙃

### CODE REVIEW

File: src/pages/Console.vue [hunk 1]
```diff
@@ -65,14 +76,9 @@ export default {
       if (this.getPopupData.show) {
         this.setPopupData({ show: false });
       }
-    }
-  },
-  watch: {
-    getLayers(value) {
-      this.loadLayers();
     },
-    getVisualizations(value) {
```
Comment (haxzie) [inline src/pages/Console.vue]: Any reason why this is removed? :smile:

Comment (rumaan) [reply src/pages/Console.vue]: Oops! Didn't quite see that. Must have deleted by mistake

**Comment by haxzie** [comment]
Thanks for the PR @rumaan :raised_hands: I have made a few changes to modularize the store and removed the mask from the console. Currently the clicks and movement from the map is being used to hide the controls :sparkles:

**Comment by rumaan** [comment]
Cools! 👍

**Comment by haxzie** [comment]
@all-contributors please add @rumaan for code

**Pull request merged**